examples/threadedExample.cpp

```diff
@@ -62,25 +66,32 @@ int main( void )
         file.first = NULL;
     }
 
-
     // Now let's test the PNG in- and output
 #   ifdef USE_PNG
         // Let's see, how the images look after several different time steps.
-        for( int i = 1; i < 33; i++)
+        file = imresh::io::readInFuncs::readPNG( "../examples/testData/atomCluster-extent-diffractionIntensity.png" );
+            imresh::libs::fftShift( file.first, file.second.first, file.second.second );
+        auto imageWidth = file.second.first;
+        auto imageHeight = file.second.second;
+        for( auto i = 1u; i <= 18; i++ )
         {
-            // How about the PNG input? BEWARE! This path is dependent on the folder structure!
-            file = imresh::io::readInFuncs::readPNG( "../examples/testData/imresh.png" );
             // Again, this step is only needed because we have no real images
-            imresh::libs::diffractionIntensity( file.first, file.second );
+            //imresh::libs::diffractionIntensity( file.first, file.second.first, file.second.second );
+            // beware diffraction intensity will be sent to the default stream thereby serializing the addTask calls!!
+            auto tmpTestImage = new float[ imageWidth * imageHeight ];
+            memcpy( tmpTestImage, file.first, imageWidth * imageHeight * sizeof( tmpTestImage[0] ) );
```

Comment:
I'm trying to delete serializing calls in order to get a nice result with nvvp. In this case `diffractionIntensity` is sent to the default stream and for normal input it isn't needed anyway, so I added a diffraction example.